### ISSUE TITLE
If SVG file has fill attribute at path tag, this attribute is not working when we set fill property on [svgStyle].

### ISSUE THREAD

**Issue by umutdeveloper** [czeckd/angular-svg-icon#76]
If SVG file has fill attribute at path tag, this attribute is not working when we set fill property on [svgStyle] because this attribute is adding fill attribute only to svg tag. But svg file has another fill propert on path tag in svg tag. 

![image](https://user-images.githubusercontent.com/20699694/60035108-bf57e900-96b4-11e9-9888-0de4e6583aa5.png)

_Originally posted by @umutcakirbm in https://github.com/czeckd/angular-svg-icon/issues/40#issuecomment-505079432_

**Comment by 0xJett** [comment]
> If SVG file has fill attribute at path tag, this attribute is not working when we set fill property on [svgStyle] because this attribute is adding fill attribute only to svg tag. But svg file has another fill propert on path tag in svg tag.
> 
> Originally posted by @umutcakirbm in #40 (comment)

I also meet this problem, have you solved it?

**Comment by CedricBojoly** [comment]
Its the same problem with the width and height.
If you have a width and height on the svg tag and add a width and height with svgStyle the icon size will not change.
Probably some svgStyles should directly be applied to the svg tag

**Comment by JV-Amorim** [comment]
Same problem here.

**Comment by czeckd** [comment]
I'm not understanding what change is being asked for here. Why not just fix the SVGs you are using?

**Comment by filipedtristao** [comment]
> I'm not understanding what change is being asked for here. Why not just fix the SVGs you are using?

Yeah, i think this is a better idea. Apply properties from the ```svg``` element to the ```path``` could be problematic, especially because you can have a svg with multiple paths.

**Comment by manga409** [comment]
> I'm not understanding what change is being asked for here. Why not just fix the SVGs you are using?

What if I'm using styles that I wanna insert programmatically, for example with an ngFor loop? I got the same svg that needs to have different colors inserted into it for each member of the list.
